cli/src/config/revsets.toml

```diff
@@ -18,7 +22,8 @@ latest(
 )
 '''
 
-'builtin_immutable_heads()' = 'trunk() | tags() | untracked_remote_bookmarks()'
+# If immutable_heads() failed to evaluate, many jj commands wouldn't work.
```

Comment:
nit: say something about `present()` here so it doesn't sound like generic explaining what `builtin_immutable_heads()` is for. Same for `revsets.log` above

Reply:
Does this work?
```
# If immutable_heads() failed to evaluate, many jj commands wouldn't work. Use
# present(expr) to suppress symbol resolution error.
```

Reply:
Yes, SGTM, thanks